src/sol/translator.rs

```diff
@@ -495,4 +496,29 @@ impl Visitor for TranslatorI {
 
         Ok(definition)
     }
+
+    fn visit_expression(
+        &mut self,
+        expression: &hir::Expression,
+    ) -> Result<Self::ExpressionOutput, Self::Error> {
+        // Use Solang Statement::Expression for future compatibility, if more
+        // expressions were to be supported
+        let definition_start = self.offset.get();
+
+        let expression_loc = self.bump(&format!(r#""{}""#, &expression.expression));
+        let string_literal = Expression::StringLiteral(vec![StringLiteral {
+            loc: expression_loc,
+            unicode: false,
+            string: expression.expression.clone(),
+        }]);
+
+        self.bump(";"); // `;` after string literal.
+
+        let definition = Statement::Expression(
+            Loc::File(0, definition_start, self.offset.get()),
+            string_literal,
+        );
```

Comment:
I see, that makes sense. But I would expect this to be more general regarding `Statement`s, instead of `Expression`s. Maybe I'm not seeing something. Could you explain the rationale to go for expressions instead the full statement?